src/modules/premium-schedules/dto/create-premium-schedule.dto.ts

```diff
@@ -0,0 +1,72 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsDateString, IsInt, IsNotEmpty, IsNumber, IsOptional, IsString, Length, MaxLength } from 'class-validator';
+
+export class CreatePremiumScheduleDto {
+  @IsInt()
+  @IsNotEmpty()
+  @ApiProperty({ example: 10588388, description: 'Can be BS or EW' })
+  readonly facilityURN: number;
+
+  @IsString()
+  @IsNotEmpty()
+  @Length(2, 2)
+  @ApiProperty({ example: 'BS', description: 'Can be BS or EW' })
+  readonly productGroup: string;
```

Comment:
If this can only be `BS` or `EW`  then should we validate this?

Reply:
Do we have example how to create custom validator?
For now I will add regex as in src\modules\exposure-period\dto\get-exposure-period-query.dto.ts